twake/backend/node/src/services/preview/src/previews/server.ts

```diff
@@ -0,0 +1,14 @@
+import PreviewService from ".";
```

Comment:
THis is not needed when putting in the node backend. Fastify is already proviuded and so you do not have to `listen`, you just have to create the routes